rajawali/src/main/java/c/org/rajawali3d/annotations/RenderThread.java

```diff
@@ -7,12 +7,12 @@
 import java.lang.annotation.Target;
 
 /**
- * Method level annotation to indicate that a method is expected to run on the GL thread.
+ * Method level annotation to indicate that a method is expected to run on the render thread.
  *
  * @author Jared Woolston (Jared.Woolston@gmail.com)
  */
 @Documented
 @Retention(RetentionPolicy.CLASS)
 @Target({ElementType.METHOD, ElementType.FIELD})
-public @interface GLThread {
+public @interface RenderThread {
```

Comment:
Yeah, I'm assuming either Android or we will provide an equivalent to GLSurfaceView and its GL Thread stuff, even if the pipeline itself uses multiple workers